### PR TITLE
JDK 24+ don't enable security manager and disable failing tests

### PR DESCRIPTION
**Pull request by theresa-m**
In [JEP 486](https://openjdk.org/jeps/486) for Java 24 any attempt to enable a security manager (such as -Djava.security.manager=allow) will result in an error. This change removes the use of java.security.manager=allow from JDK 24+.

The system test framework also has uses the security manager for load tests that needs to be removed. In the meantime this change disables tests that are failing because of it. See https://github.com/adoptium/STF/issues/142

### CODE REVIEW

**Comment by theresa-m** [comment]
The remaining failures are due to use of the SecurityManager library in the system test framework. I opened https://github.com/adoptium/STF/issues/142 and will temporarily disable the failing tests.

**Comment by theresa-m** [comment]
This is ready for review fyi @llxia

**Review by llxia — APPROVED** [review]
LGTM

File: system/system.mk [hunk 1]
```diff
@@ -47,14 +47,15 @@ else
   OPENJ9_PRAM=""
 endif
 
+# In JDK 24+ any attempts to enable the security manager will result in an error
```
Comment (pshipton) [inline system/system.mk:50]: This sentence is missing a period.

**Review by pshipton — CHANGES_REQUESTED** [review]

File: system/system.mk (hunk 1 above)
Comment (theresa-m) [reply system/system.mk]: Fixed.

**Review by pshipton — APPROVED** [review]

**Review by smlambert — APPROVED** [review]
👍

**Pull request merged**